libxrdp/xrdp_rdp.c

```diff
@@ -222,18 +222,8 @@ xrdp_rdp_read_config(struct xrdp_client_info *client_info)
         }
         else if (g_strcasecmp(item, "key_file") == 0)
         {
-            g_memset(client_info->key_file, 0, sizeof(char) * 1024);
-            if (g_strlen(value) == 0)
```

Comment:
> This change makes the default configuration invalid, so it's not good.
I do not understand.
The default values are set above. before this change they was not set if ```key_file``` was absent, ```key_file=``` was required to set the default value.